tests/test_compute_metrics.py

```diff
@@ -0,0 +1,33 @@
+from dscigametrics.compute_metrics import compute_metrics
+import pytest
+import pandas as pd
+
+df = pd.read_csv('ga_metrics_test_data.csv')
```

Comment:
Since we are running the command `pytest tests/` under the directory of our repo (which is `Group_9_GA_Metrics/`), this path should be changed to`'tests/ga_metrics_test_data.csv'`. Otherwise, the machine will not be able to find the test data.

```suggestion
df = pd.read_csv('tests/ga_metrics_test_data.csv')
```

Reply:
Yes! Cindy mentioned this in office hours too, well spotted!